modules/ROOT/pages/functions/vector.adoc

```diff
@@ -0,0 +1,147 @@
+:description: Vector functions allow you to compute the similarity scores of vector pairs.
+
+:link-vector-indexes: xref:indexes/semantic-indexes/vector-indexes.adoc
+
+[[query-functions-vector]]
+= Vector functions
+
+Vector functions allow you to compute the similarity scores of vector pairs.
+These vector similarity functions are identical to those used by Neo4j's {link-vector-indexes}[vector search indexes].
+
+Consider for example _k_-nearest neighbor queries, which return the _k_ entities with highest similarity scores based on comparing their associated vectors with a query vector.
+Such queries can be run against vector indexes to quickly get an approximate result.
```

Comment:
That's fair. I wanted to avoid restating too much from the vector index documentation, but I'll see if I can clarify this bit.

It's not the score that's approximate; it's whether or not the true nearest neighbours are included in the result set.